_components/pages/History.tsx

```diff
@@ -0,0 +1,213 @@
+import * as React from "react";
+import { usePathPrefix } from '@redocly/developer-portal/ui';
+
+export default function History() {
+  const prefix = usePathPrefix()
+  return (
+    <div className="overflow-hidden landing">
```

Comment:
In light mode, I'm seeing white-on-white text in this page that I think can be fixed by adding this class:

```suggestion
    <div className="overflow-hidden landing styled-page">
```

(Same cause and same solution as I encountered on the XRPL overview and as I spotted on the Use Cases page—explained in slightly more detail there.)

Reply:
added, waiting for @JST5000 to give me access I'll then commit the changes